docs/debug_faq.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Why is Tilt broken?
+layout: docs
+---
+
+Tilt is broken. What should you do?
+
+There are a lot of ways to get help, depending on your willingness to get your
```

Comment:
Depending on _how_ willing you are to get your hands dirty?

Reply:
done!